manifests/apis/webhooks/kustomization.yml

```diff
@@ -1,9 +1,44 @@
+namespace: rukpak-system
+namePrefix: rukpak-
+
 resources:
-  - 00_namespace.yaml
-  - 01_cluster_role.yaml
-  - 02_role.yaml
-  - 03_serviceaccount.yaml
-  - 04_cluster_role_binding.yaml
-  - 05_role_binding.yaml
-  - 06_deployment.yaml
-  - 07_webhook.yaml
+  - resources/namespace.yaml
+  - resources/serviceaccount.yaml
+  - resources/service.yaml
+  - resources/certificate.yaml
+  - resources/deployment.yaml
+  - resources/webhook.yaml
+
+configurations:
+- kustomizeconfig.yaml
+
+patchesStrategicMerge:
+- patches/cainjection.yaml
+
+vars:
+- name: CERTIFICATE_NAMESPACE # namespace of the certificate CR
+  objref:
+    kind: Certificate
+    group: cert-manager.io
+    version: v1
+    name: webhook-certificate # this name should match the one in certificate.yaml
+  fieldref:
+    fieldpath: metadata.namespace
```

Comment:
This defines the variable `CERTIFICATE_NAMESPACE`, which is derived from the `metadata.namespace` value of the `cert-manager.io/v1 Certificate` called `webhook-certificate`.